Dockerfile-linux.template

```diff
@@ -516,6 +516,11 @@ RUN set -eux; \
 
 COPY docker-php-ext-* docker-php-entrypoint /usr/local/bin/
 
+{{ if rcVersion | IN("8.1", "8.2", "8.3", "8.4") then ( -}}
+# enable OPcache by default (https://wiki.php.net/rfc/make_opcache_required)
+RUN docker-php-ext-enable opcache
+{{ ) end -}}
```

Comment:
Gah, `if` without `else` isn't supported until `jq` 1.7 (https://github.com/jqlang/jq/releases/tag/jq-1.7), which we don't have on our primary infra (Debian-based, thus still `jq` 1.6).  I've fixed it in a follow-up commit, but that's something we'll need to be aware of and watching for now (or will have to update CI across _all_ our repos to deal with somehow). :disappointed:

(If only GHA let us provide our own VM images without having to manage the VMs ourselves too :sob:)